pkg/application/app.go

```diff
@@ -625,8 +616,11 @@ func (app *Avalanche) ConfigFileExists() bool {
 	return true
 }
 
-func (app *Avalanche) WriteConfigFile(bytes []byte) error {
+func (app *Avalanche) WriteConfigFile(bytes []byte, path string) error {
 	configPath := app.GetConfigPath()
+	if path != "" {
+		configPath = path
+	}
 	return app.writeFile(configPath, bytes)
 }
```

Comment:
add a function app.LoadConfig and app.SetConfigValue, probably another app.GetConfigValue

Reply:
I believe we can also have a new Config struct for this, encapsulating viper, with an instance of it on app